Ports/qoi/patches/0002-Add-musl-libc-frexp-implementation-for-stb_image_wri.patch

```diff
@@ -0,0 +1,53 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Liav A <liavalb@gmail.com>
+Date: Sun, 26 Jun 2022 22:25:21 +0300
+Subject: [PATCH] Add musl-libc frexp implementation for stb_image_write header
+ file
+
+---
+ stb_image_write.h | 25 +++++++++++++++++++++++++
+ 1 file changed, 25 insertions(+)
+
+diff --git a/stb_image_write.h b/stb_image_write.h
+index d864be7..0905d00 100644
+--- a/stb_image_write.h
++++ b/stb_image_write.h
+@@ -153,6 +153,8 @@ LICENSE
+ #define INCLUDE_STB_IMAGE_WRITE_H
+ 
+ #include <stdlib.h>
++#include <math.h>
++#include <stdint.h>
+ 
+ // if STB_IMAGE_WRITE_STATIC causes problems, try defining STBIWDEF to 'inline' or 'static inline'
+ #ifndef STBIWDEF
+@@ -637,6 +639,29 @@ STBIWDEF int stbi_write_tga(char const *filename, int x, int y, int comp, const
+ 
+ #ifndef STBI_WRITE_NO_STDIO
+ 
++// FIXME: This is an implementation from musl libc to help other code to be able to function
++// correctly. Remove this once we implement this in our math library.
```

Comment:
Without looking too closely at the implementation here, this seems like a function that isn't too hard to write a naive implementation for. I don't think there is any precedent, but I guess the "not invented here" rule applies for function replacements in ports patches as well.